makefile

```diff
@@ -1,21 +1,25 @@
 NUM_NODES = 10
 WORLD_SIZE = 1000
 
-COMMON_CFLAGS = -g -std=gnu99 -O2 -mcpu=native -fomit-frame-pointer -Wall -Wextra
+COMMON_CFLAGS = -std=gnu99 -O2 -march=native -fomit-frame-pointer -Wall -Wextra
```

Comment:
But previously the makefile didn't even specify `-mtune`. `-march` at least implies `-mcpu`.